cg/cli/workflow/microsalt/base.py

```diff
@@ -179,6 +179,8 @@ def start(
 ) -> None:
     """Start whole microSALT workflow by providing case, ticket or sample id"""
     LOG.info("Starting Microsalt workflow for %s", unique_id)
+    if not sample and not ticket:
+        context.invoke(resolve_compression, case_id=unique_id, dry_run=dry_run)
```

Comment:
This falls outside of the context of the PR. Is the inclusion intended?

Reply:
I noticed I missed it in my refactor. Nah it's not needed but I can move it to its own pr later if you want